trino/trino_test.go

```diff
@@ -62,6 +62,21 @@ func TestConfigSSLCertPath(t *testing.T) {
 	assert.Equal(t, want, dsn)
 }
 
+func TestConfigSSLCert(t *testing.T) {
+	c := &Config{
+		ServerURI:         "https://foobar@localhost:8080",
+		SessionProperties: map[string]string{"query_priority": "1"},
+		SSLCert:           "cert.pem",
```

Comment:
`SSLCert` should have contents of a cert, it's not a file path like `SSLCertPath`. This is important to test to make sure that a large value with equal signs (base64 encoded) and newlines won't cause any issues here.

Reply:
@mrveera are you going to continue working on this PR? If not, I can handle this.

Reply:
I opened https://github.com/trinodb/trino-go-client/pull/77 with comments addressed.